checkstyle/checkstyle.xml

```diff
@@ -60,10 +60,9 @@
     <module name="TypeName"/>
     
     <!-- whitespace -->
-    <module name="GenericWhitespace"/>
```

Comment:
I believe this check is buggy for handling collections with templatized, non-static inner classes. 

`PriorityQueue<TriggerContext<WK>.TriggerCallbackState<WK>>`